src/utils/registerInterfaceWithDefaultTheme.js

```diff
@@ -0,0 +1,7 @@
+import ThemedStyleSheet from 'react-with-styles/lib/ThemedStyleSheet';
+import DefaultTheme from '../themes/DefaultTheme';
+
+export default function registerInterfaceWithDefaultTheme(reactWithStylesInterface) {
```

Comment:
it'd be good to change the test setup and storybook config to use this instead maybe